CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.6)
 
-project(LAPACK)
+project(LAPACK Fortran C)
```

Comment:
With the project language specified here, statements `enable_language(Fortran)` at lines 226 and 301 are not needed anymore.